Modules/_abc.c

```diff
@@ -673,7 +674,7 @@ compute_abstract_methods(PyObject *self)
             Py_DECREF(base_abstracts);
             goto error;
         }
-
+        Py_DECREF(base_abstracts);
         PyObject *key, *value;
         while ((key = PyIter_Next(iter))) {
```

Comment:
Check for iterators that raise an error is still missing

Reply:
There is `if PyErrorOcurred()` below, it will betrigerred if the loop is exited on execption.

Reply:
You're right, I missed that